### ISSUE TITLE
accessibility_tools reports itself as being inaccessible

### ISSUE THREAD

**Issue by paulking86** [rebelappstudio/accessibility_tools#71]
I have noticed that when pressing the FAB to bring up the UI Settings overlay, the accessibility_tools package reports itself for having multiple accessibility issues!

```
I/flutter ( 1372): ==========================
I/flutter ( 1372): ACCESSIBILITY ISSUES FOUND
I/flutter ( 1372): ==========================
I/flutter ( 1372): 
I/flutter ( 1372): Accessibility issue 1: Tap area of 81.49x44.29 is too small:
I/flutter ( 1372): should be at least 48x48
I/flutter ( 1372): 
I/flutter ( 1372): The relevant error-causing widget was:
I/flutter ( 1372):   ChoiceChip
I/flutter ( 1372):   ChoiceChip:file:///Users/paulking/.pub-cache/hosted/pub.dev/accessibility_tools-2.4.0/lib/src/testing_tools/multi_value_toggle.dart:48:13
I/flutter ( 1372): 
I/flutter ( 1372): 
I/flutter ( 1372): Consider making the tap area bigger. For example, wrap the widget in a SizedBox:
I/flutter ( 1372): 
I/flutter ( 1372): InkWell(
I/flutter ( 1372):   child: SizedBox.square(
I/flutter ( 1372):     dimension: 48,
I/flutter ( 1372):     child: child,
I/flutter ( 1372):   ),
I/flutter ( 1372): )
I/flutter ( 1372): 
I/flutter ( 1372): Icons have a size property:
I/flutter ( 1372): 
I/flutter ( 1372): Icon(
I/flutter ( 1372):   Icons.wysiwyg,
I/flutter ( 1372):   size: 48,
I/flutter ( 1372): )
I/flutter ( 1372): 
I/flutter ( 1372): 
I/flutter ( 1372): Accessibility issue 2: Tap area of 49.94x44.29 is too small:
I/flutter ( 1372): should be at least 48x48
I/flutter ( 1372): 
I/flutter ( 1372): The relevant error-causing widget was:
I/flutter ( 1372):   ChoiceChip
I/flutter ( 1372):   ChoiceChip:file:///Users/paulking/.pub-cache/hosted/pub.dev/accessibility_tools-2.4.0/lib/src/testing_tools/multi_value_toggle.dart:55:22
I/flutter ( 1372): 
I/flutter ( 1372): 
I/flutter ( 1372): Consider making the tap area bigger. For example, wrap the widget in a SizedBox:
I/flutter ( 1372): 
I/flutter ( 1372): InkWell(
I/flutter ( 1372):   child: SizedBox.square(
I/flutter ( 1372):     dimension: 48,
I/flutter ( 1372):     child: child,
I/flutter ( 1372):   ),
I/flutter ( 1372): )
I/flutter ( 1372): 
I/flutter ( 1372): Icons have a size property:
I/flutter ( 1372): 
I/flutter ( 1372): Icon(
I/flutter ( 1372):   Icons.wysiwyg,
I/flutter ( 1372):   size: 48,
I/flutter ( 1372): )
I/flutter ( 1372): 
I/flutter ( 1372): 
I/flutter ( 1372): Accessibility issue 3: Tap area of 52.13x44.29 is too small:
I/flutter ( 1372): should be at least 48x48
I/flutter ( 1372): 
I/flutter ( 1372): The relevant error-causing widget was:
I/flutter ( 1372):   ChoiceChip
I/flutter ( 1372):   ChoiceChip:file:///Users/paulking/.pub-cache/hosted/pub.dev/accessibility_tools-2.4.0/lib/src/testing_tools/multi_value_toggle.dart:55:22
I/flutter ( 1372): 
I/flutter ( 1372): 
I/flutter ( 1372): Consider making the tap area bigger. For example, wrap the widget in a SizedBox:
I/flutter ( 1372): 
I/flutter ( 1372): InkWell(
I/flutter ( 1372):   child: SizedBox.square(
I/flutter ( 1372):     dimension: 48,
I/flutter ( 1372):     child: child,
I/flutter ( 1372):   ),
I/flutter ( 1372): )
I/flutter ( 1372): 
I/flutter ( 1372): Icons have a size property:
I/flutter ( 1372): 
I/flutter ( 1372): Icon(
I/flutter ( 1372):   Icons.wysiwyg,
I/flutter ( 1372):   size: 48,
I/flutter ( 1372): )
```

I guess the ideal solution would be to ensure that the UI is adjusted to make sure it is fully accessible, but at the very least maybe these issues could be silenced as to not potentially confuse people? This occurs for me on an Google Pixel 7 pro running on Android 15 👍

**Comment by aednlaxer** [comment]
Hi Paul, thanks for reporting this issue. Indeed, the best solution would be to fix those accessibility issues. I'll take a look!

**Comment by aednlaxer** [comment]
I've tested accessibility tools in various conditions. At first I wasn't able to reproduce the issue but after fiddling with Android's bouncing scrolling physics I saw some false positive issues reported. Accessibility tools uses Material widgets that are accessible, accessibility testing panel is widget-tested to be accessible, too. However, in some cases when scrolling or when min tap area guideline is large several issues might be reported.

I didn't find actual accessibility issues to fix so I made a change to ignore min tap area checks inside accessibility tools. This behaviour is disabled in tests so accessibility tools are still tested to be accessible.

This change has been released in 2.4.1. Hope it fixes the problem!

**Comment by paulking86** [comment]
Can confirm that this resolves the issue @aednlaxer - Thank you for the rapid turnaround 👏
